src/components/Navigation.js

```diff
@@ -1,42 +1,72 @@
 import React from 'react';
 import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 
-export default function Navigation() {
+export function Navigation() {
     return(
         <HeaderStyled>
             <h2>Event Planner</h2>
-            {/* <img class="menu-button" src="https://cdn3.iconfinder.com/data/icons/mobile-friendly-ui/100/menu-512.png" ></img> */}
                <MenuStyled>
                 <ul>
-                    <li>home</li>
-                    <li>about</li>
-                    <li>log in</li>
-                    <li>sign up</li>
+                    <Link className="link" to='/'>Home</Link>
+                    <Link className="link" to='/'>About</Link>
+                    <Link className="link " to='./login'>Log In</Link>
+                    <Link className="link " to='./signup'>Sign up</Link>
                 </ul>
                 </MenuStyled>
         </HeaderStyled>
     )
 }
+
+export function NavigationLoggedin(){
```

Comment:
You don't need a separate component. You can handle the conditional rendering inside Navigation above.